app/utils/wikiQuery.tsx

```diff
@@ -1,45 +1,20 @@
+import { IWikipediaContestantData } from './wikiFetch'
 
-import * as cheerio from 'cheerio'
-
-export const wikiUrl = "https://en.wikipedia.org/wiki/The_Amazing_Race_35"
-
-interface IWikipediaData {
-    parse: {
-        text: string
-    }
-}
-
-export interface IContestant {
+export interface ITeam {
     teamName: string
-    age: number
     relationship: string
-    hometown: string
     isParticipating: boolean
     eliminationOrder: string
 }
 
-async function fetchWikipediaData(): Promise<IWikipediaData> {
-    const apiUrl = 'https://en.wikipedia.org/w/api.php?action=parse&format=json&page=The_Amazing_Race_35&section=7&formatversion=2'
-    const response = await fetch(apiUrl)
-    const data = await response.json()
-    return data
-}
-
-export async function getContestantList(): Promise<any> {
+export function getTeamList(contestantData :IWikipediaContestantData[]): any {
```

Comment:
We could probably change `any` here to `ITeam[]`.

Reply:
The issue with that is it still returns a `return { props: { runners: contestants }}` which is to say it returns some object with a props object which has a runners property which is a `ITeam[]` 😅 and TBH I was just a tad too lazy to create interfaces for all those. But really there is nothing stopping us, you are right!

Reply:
Similar to the above, I going to opt to not do this at the moment, but I do think we should keep this in mind because relying on the any type here is not good!